README.md

```diff
@@ -165,7 +165,7 @@ pub struct MyCredentialSubject {
   email: String
 }
 
-let credential = SpecializedJsonCredential::<MyCredentialSubject>::new(
+let credential = ssi::claims::vc::v1::JsonCredential::<MyCredentialSubject>::new(
```

Comment:
Does it mean there's no type to be able to accept both versions now?

Reply:
I realise it's because the version is a generic parameter. I would like to have maybe an enum to be able to support both versions out of the box.

Reply:
Yes I agree I should add a type that can accept both versions. Also I should add a type that accepts any Data-Intgrity-compatible document, VC or not.

Reply:
I've added the `AnyJsonCredential` and `AnyJsonPresentation` types that combine v1 and v2. I've also added a `DataIntegrityDocument` representing any Data-Integrity-compatible document, even if its not a VC.